Remotion/ObjectBinding/Web/Globalization/BocList.de.resx

```diff
@@ -124,7 +124,7 @@
     <value>Ansicht</value>
   </data>
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.RowEditModeErrorMessage" xml:space="preserve">
-    <value>In der aktuellen Zeile sind Fehler aufgetreten.</value>
+      <value>Ungültige Eingabe in der aktuellen Zeile</value>
```

Comment:
```suggestion
    <value>Ungültige Eingabe in der aktuellen Zeile</value>
```

pleas echeck your resx. soemthing changed the indent on some of the entries. since I can't say if that's a setting on your system, just because the changes where made via Rider-Localization UI or due to some other unknown reason, it's best to fix them since there is no expectation that other developers wouldn't revert them to the original two-whitespace indent.